src/Mvc/Mvc.Core/test/ControllerBaseTest.cs

```diff
@@ -3056,11 +3081,11 @@ private static ControllerBase GetController(IModelBinder binder, IValueProvider
                     stringLocalizerFactory: null),
             };
 
-            valueProvider = valueProvider ?? new SimpleValueProvider();
+            valueProvider ??= new SimpleValueProvider();
```

Comment:
+32 PranavPoints.

Reply:
How can I get those (and can I exchange them for Schrute bucks)?